<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopNewCaliProduct</class>
 <widget class="QDialog" name="PopNewCaliProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>测试模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>360</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设置校准品默认位置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>135</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>项目名称：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>校准品批号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>392</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>C0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>C1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>470</y>
     <width>434</width>
     <height>47</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_NewCaliReset">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}</string>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_NewCaliOK">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_NewCaliCancle">
      <property name="minimumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>140</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnNomal.png);
padding: -1;}
QPushButton:pressed{
font: 700 13pt &quot;Microsoft YaHei UI&quot;;
color: #F2F2F2;
border-image: url(:/resources/img/common/btnClicked.png);
padding: -1;}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
	background-color: rgb(0, 48, 145);
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
 font: 700 12pt &quot;Microsoft YaHei UI&quot;;
</string>
      </property>
      <property name="text">
       <string>校准品</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
